coursemapper-kg/recommendation/app/services/course_materials/recommendation/recommender.py

```
@@ -39,157 +43,116 @@ def sort_by_similarity_type(data, similarity_type):
-    def recommend(
-        self,
-        not_understood_concept_list,
-        slide_concepts=[],
-        slide_weighted_avg_embedding_of_concepts="",
-        slide_document_embedding="",
-        user_embedding="",
-        top_n=10,
-        video=True,
-        recommendation_type=RecommendationType.WITHOUT_EMBEDDING,
-    ):
+    def _get_data(self, recommendation_type, not_understood_concept_list, slide_concepts, video):
```

Ruff (0.12.2)

Missing return type annotation for private function `_get_data`

(ANN202)